setup.py

```diff
@@ -38,7 +38,13 @@
 # Add S3QL sources
 sys.path.insert(0, os.path.join(basedir, 'src'))
 sys.path.insert(0, os.path.join(basedir, 'util'))
-import s3ql
+
+# Get S3QL Version & Release from __init__.py
+with open('src/s3ql/__init__.py', 'r') as init_file:
+    init_content = init_file.read()
+    VERSION = re.compile("^VERSION\s*=\s*'(\d+\.\d+)'", re.MULTILINE).search(init_content).groups(0)[0]
```

Comment:
Why the `compile` call? You can just use re.search..

Reply:
I see, done.